CS3219-AY2223S1/cs3219-project-ay2223s1-g12
# Implement Authorizations to backend and frontend API with cookies

Fixes #69 
Depends on #63, #68

**Summary**
* Implement protection of /logout route, both in frontend and backend
* Update protection to use cookies instead of authorization bearer token
* Sync cookie expiration time to that of their jwt token counterparts
* Implement redirection to login if token/cookie expired

**Steps to test**
1. Navigate to `http://localhost:3000/login` and login
2. Upon redirection to /home, wait for 30s for the access token cookie to expire, then click on the logout button.
3. Observe that the logout fails and redirects you to the login page again
4. For phase 2: In this current state (with no cookie by the name of 'token' in the browser), forcibly navigate to `http://localhost:3000/logout`
5. Observe that the frontend warns that "You must be logged in first!"
